pom.xml

```diff
@@ -1,79 +1,87 @@
-	<description>A fork or the Phil Varner library to interact with clamd for virus scanning.</description>
-	<url>http://philvarner.com/clamavj/</url>
-	<licenses>
-		<license>
-			<name>Apache Software License, 2.0</name>
-			<url>http://www.apache.org/licenses/LICENSE-2.0</url>
-		</license>
-	</licenses>
-
-	<distributionManagement>
-		<repository>
-			<id>dlbuild.libint.unc.edu</id>
-			<name>dlbuild.libint.unc.edu-releases</name>
-			<url>http://dlbuild.libint.unc.edu:8080/artifactory/libs-release-local</url>
-		</repository>
-		<snapshotRepository>
-			<id>dlbuild.libint.unc.edu</id>
-			<name>dlbuild.libint.unc.edu-snapshots</name>
-			<url>http://dlbuild.libint.unc.edu:8080/artifactory/libs-snapshot-local</url>
-		</snapshotRepository>
-	</distributionManagement>
-
-	<reporting>
-		<plugins>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-project-info-reports-plugin</artifactId>
-				<reportSets>
-					<reportSet>
-						<reports>
-							<report>dependencies</report>
-							<report>license</report>
-							<report>scm</report>
-							<report>project-team</report>
-							<report>index</report>
-						</reports>
-					</reportSet>
-				</reportSets>
-			</plugin>
-		</plugins>
-	</reporting>
-	<build>
-		<pluginManagement>
-			<plugins>
-				<plugin>
-					<artifactId>maven-compiler-plugin</artifactId>
-					<configuration>
-						<source>1.6</source>
-						<target>1.6</target>
-					</configuration>
-				</plugin>
-				<plugin>
-					<artifactId>maven-javadoc-plugin</artifactId>
-				</plugin>
-			</plugins>
-		</pluginManagement>
-	</build>
-	<dependencies>
-		<dependency>
-			<groupId>commons-logging</groupId>
-			<artifactId>commons-logging</artifactId>
-			<version>1.1</version>
-			<scope>provided</scope>
-		</dependency>
-		<dependency>
-			<groupId>junit</groupId>
-			<artifactId>junit</artifactId>
-			<version>4.10</version>
-			<scope>test</scope>
-		</dependency>
-	</dependencies>
+    <name>ClamAV Daemon Java API</name>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>edu.unc.lib.cdr</groupId>
+    <artifactId>clamavj</artifactId>
+    <version>0.3</version>
+    <description>A fork or the Phil Varner library to interact with clamd for virus scanning.</description>
+    <licenses>
+        <license>
+            <name>Apache Software License, 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0</url>
+        </license>
+    </licenses>
+    <reporting>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-project-info-reports-plugin</artifactId>
+                <reportSets>
+                    <reportSet>
+                        <reports>
+                            <report>dependencies</report>
+                            <report>license</report>
+                            <report>scm</report>
+                            <report>project-team</report>
```

Comment:
remove blank line?

Reply:
I don't think there is a blank line, its just a github line wrapping issue